GitHub Actions/haal-centraal-npm-beheer.md

```diff
@@ -160,6 +160,11 @@ Wil je met één statement alle packages updaten naar de meest recente versie, d
 - `ncu -u`. Met deze statement worden de npm package referenties in een package.json bestand ge-update naar de meest recente versie
 - `npm install`. Met deze statement worden de packages in de node_modules bestand geupdate aan de hand van het package.json bestand
```

Comment:
ik zie hier een typo: node_modules **bestand** moet zijn node_modules **folder of map**